src/services/smart-fields-values-injector.js

```diff
@@ -90,7 +105,13 @@ function SmartFieldsValuesInjector(record, modelName, fieldsPerModel, depth = 0)
               return null;
             }
 
-            if (!record[field.field][fieldAssociation.field]
+            if (record
+              && record[field.field]
+              && record[field.field].dataValues
+              && !Object.prototype.hasOwnProperty.call(
+                record.dataValues[field.field].dataValues,
```

Comment:
```suggestion
                record.getDataValue(field.field).get(),
```

Reply:
It feel very weird to me to call `hasOwnProperty` on `.get()` instead of `.dataValues`